test/actions/mintNFT.test.ts

```diff
@@ -0,0 +1,94 @@
+import { Keypair, PublicKey } from '@solana/web3.js';
+import axios, { AxiosResponse } from 'axios';
+import { Connection } from './../../src/Connection';
+import { NodeWallet, Wallet } from './../../src/wallet';
+import { mintNFT } from './../../src/actions';
+import { FEE_PAYER, MASTER_EDITION_PUBKEY } from './../utils';
+import { Token } from '@solana/spl-token';
+import { MasterEdition, Metadata } from './../../src/programs/metadata';
+
+jest.mock('axios');
+
+const mockedAxiosGet = axios.get as jest.MockedFunction<typeof axios>;
+
+const METADATA_JSON_URL =
+  'https://bafkreibj4hjlhf3ehpugvfy6bzhhu2c7frvyhrykjqmoocsvdw24omfqga.ipfs.dweb.link';
+const MINT_TRANSACTION_ID =
+  '64Tpr1DNj9UWg1P89Zss5Y4Mh2gGyRUMYZPNenZKY2hiNjsotrCDMBriDrsvhg5BJt3mY4hH6jcparNHCZGhAwf6';
+const ASSOCIATED_TOKEN_ACCOUNT_PUBKEY = new PublicKey(
+  '6rUp1UcrKrtxsEEwBSLCByPLAkcFvqrAzp7TmJj5cHpm',
+);
+const METADATA_PDA_PUBKEY = new PublicKey('GCSH5P3d37Meoc9aU8xAuEM8q3P2FfF9UMTbAxFCT19z');
+const EDITION_PDA_PUBKEY = new PublicKey('GCSH5P3d37Meoc9aU8xAuEM8q3P2FfF9UMTbAxFCT19z');
```

Comment:
I'm pulling these mocked values from a test case I ran while the connection was still allowed to go through to devnet.

https://explorer.solana.com/address/GCSH5P3d37Meoc9aU8xAuEM8q3P2FfF9UMTbAxFCT19z?cluster=devnet

I could use a double-check on the public keys that I looked up are correct. Right now both using what I believe is the master edition pda.